src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServices.feature

```diff
@@ -0,0 +1,39 @@
+﻿Feature: Additional Services
+	As a Buyer
+	I want to manage the Additional Services of Order Form
+	So that I can ensure the information is correct
+
+Scenario: Additional Services - Sections presented
+Given an unsubmitted order exists
+When the Order Form for the existing order is presented
+Then there is the Additional Service section
+And the User is able to manage the Additional Services section
+
+Scenario: Additional Services - Catalogue Solution now complete, >= 1 Service Recipient, >=1 Catalogue Solution
+Given an unsubmitted order exists
+When the Order Form for the existing order is presented
+Then the Call Off Agreement ID is displayed
+And there is the Order description section
+And there is the Call-off Ordering Party section
+And there is the Supplier section
+And there is the Commencement date section
+And there is the Service Recipients section
+And there is the Additional Service section
+And the Preview order summary button is enabled
+And the Delete order button is enabled
+And the Submit order button is disabled
+
+Scenario: Additional Services - Catalogue Solution now complete, >= 1 Service Recipient, 0 Catalogue Solution
+Given an unsubmitted order exists
```

Comment:
can you move this data setup into a background please?

Reply:
Done